setup.py

```diff
@@ -281,6 +282,7 @@ def do_setup():
             'oracle': oracle,
             'password': password,
             'postgres': postgres,
+            'pydruid': druid,
```

Comment:
NIT: I'd go with `druid` instead of `pydruid`, the idea being that the extra_requires represents a set of libs needed to enable a hook. If the Druid integration grows to require more pypi packages the name `druid` will still make sense, and `pydruid` would just be one of the deps in that extra.